R/plotting.R

```diff
@@ -33,6 +33,7 @@ plot.disag.data <- function(x, ...) {
 
 plot.fit.result <- function(x, ...){
   
+  parameter <- sd <- obs <- pred <- NULL
```

Comment:
What does this line do?

Reply:
This stops the warning in R CMD check (or `check()`). Those objects are used in non standard evaluation in ggplot or dplyr calls. Like `ggplot(d, aes(x, y))`. `x` and `y` aren't actually objects but the R check tests whether all used objects exist.

 I think there's a more official way to do it but I can't remember.